### ISSUE TITLE
Issue with passing empty organizationID variable

### ISSUE THREAD

**Issue by AyalaJuan** [simple-salesforce/simple-salesforce#161]
I noticed that when only passing username and password 
`simple_salesforce.Salesforce(username=self.user,
                                                password=self.passwd,
                                                )`

I receive the error "You must provide login information or an instance and token".

Yet when I provide the variable organizationId, it goes through fine.
`simple_salesforce.Salesforce(username=self.user,
                                                password=self.passwd,
                                                organizationId='')`

Reading through login.py , the only difference in between the two authentication methods is the organizationID being formatted into the body. If I'm passing an empty organization ID I don't see why it works. Maybe it's the salesforce API that is wack?

**Comment by andscoop** [comment]
Appears to be related to #171 , we are potentially being too strict with our requirements for SalesforceLogin() across all use cases.

**Comment by gokumatrix** [comment]
This workaround didn't work for me, and I'm still getting the INVALID_LOGIN. Is there another workaround I can try besides passing a blank organizationID?

**Comment by jamesward** [comment]
Yeah, `Salesforce` is more strict that `SalesforceLogin` which allows:
```
SalesforceLogin(username=salesforce_username, password=salesforce_password)
```
